core/modules/user/user.theme.inc

```diff
@@ -172,15 +172,18 @@ function theme_user_permission_description($variables) {
 function theme_user_admin_roles($variables) {
   $form = $variables['form'];
 
-  $header = array(t('Name'), t('Weight'), t('Operations'));
+  $header = array(t('Name'), t('Required'), t('Weight'), t('Operations'));
   foreach (element_children($form['roles']) as $role_name) {
-    $label = $form['roles'][$role_name]['#role']->label;
     $row = array();
+    $row[] = theme('label_machine_name__role', array(
+      'label' => $form['roles'][$role_name]['#role']->label,
+      'machine_name' => $form['roles'][$role_name]['#role']->name,
+    ));
     if (in_array($role_name, array(BACKDROP_ANONYMOUS_ROLE, BACKDROP_AUTHENTICATED_ROLE))) {
-      $row[] = t('@name <em>(required)</em>', array('@name' => $label));
+      $row[] = t('yes');
```

Comment:
Should this be captialized?

Reply:
Not too fussed about it TBH. I would really like us to change this to a useful description instead (https://github.com/backdrop/backdrop-issues/issues/5179), which would replace this anyway.

Reply:
I don't think this is a pattern we've used elsewhere of saying "yes" in a column for requiredness:

![image](https://user-images.githubusercontent.com/100206/131265158-d4989a79-d05d-4689-9867-bb17e8823085.png)

Just using "yes" seems vague to me what it means.

Reply:
As discussed with @quicksketch in the sprint zoom room, the preference here is to revert to adding the `(required)` bit as a suffix to the role name, and remove the "Required" column altogether. In a follow up like https://github.com/backdrop/backdrop-issues/issues/5179 we can add the ability to provide descriptions for roles, and have the descriptions for Anonymous/Authenticated be "locked" or hardcoded to the description text we provide.

Doing that next...

Reply:
Done. Side-by-side before/after:

![image](https://user-images.githubusercontent.com/2423362/131266533-32426ae7-cd20-4816-b9d4-6978eab94c08.png)